info_extraction/src/edu/gatech/gtri/info_extraction/utils.py

```diff
@@ -0,0 +1,125 @@
+from collections import defaultdict
```

Comment:
This is not java, there is no need for this enterprise grade file path.

Reply:
maybe just `/extractors/src/jlextractor.py` is sufficient?

Reply:
That sounds like a subjective things between individuals; everyone has their own naming habits; its your repo so feel free to change to whatever you'd like :)

Reply:
It is not a subjective thing between individuals, you are following a standard conventions from Java that is not consistent with either python or julia naming conventions.

Reply:
Oh no, I wasn't thinking of java when I did it. I was thinking of python packages. Python has nested packages. edu.gatech.gtri.info_extraction is a valid python nested package specification; it may not be an individual's preference. Since it is python code, I do recommend we stay with a python package, not just a collection of python scripts.  would semanticmodels.jl.info_extraction work? The top-level capture the project name and the 2nd level captures the module for information extraction.